src/index.ts

```diff
@@ -547,8 +542,9 @@ export class Redlock extends EventEmitter {
     // of a failure to extend the lock, and subsequent expiration. In the event
     // of an abort, the error object will be made available at `signal.error`.
     const controller =
-      typeof AbortController === 'undefined'
-        ? new PolyfillAbortController()
+        // AbortController became available as a global in node version 16. Once version 14 reaches its end-of-life, this can be removed.
+        typeof AbortController === 'undefined'
+        ? new (await import('node-abort-controller')).AbortController()
```

Comment:
The inline await is a bit "overly punctuated." Maybe a function getAbortController?

Reply:
removed this code